sdks/java/io/elasticsearch-tests/elasticsearch-tests-common/src/test/java/org/apache/beam/sdk/io/elasticsearch/ElasticsearchIOTestCommon.java

```diff
@@ -111,7 +124,7 @@ static String getEsIndex() {
   }
 
   static final String ES_TYPE = "test";
-  static final long NUM_DOCS_UTESTS = 100L;
+  static final long NUM_DOCS_UTESTS = 40L;
```

Comment:
Just out of interest - why, please?

Reply:
This was a bit of grasping at straws to be honest. My thought was that fewer docs would hopefully lower the flakiness of the uTests given that it would be less arduous on the state to write fewer docs. But granted, 100 is not so different from 40.